pkg/config/config.go

```diff
@@ -102,3 +111,43 @@ func (c *Config) getVfInfo(vfPci string) (string, int, error) {
 
 	return pf, vfID, nil
 }
+
+// handleBridgeConfig checks PF bridge configuration and set PFBridge options for PluginConfig.
+// the function checks that PF is attached to one of the bridges which explicitly set
+// in the configuration or to the default bridge, returns error if PF is not a part of a bridge
+func (c *Config) handleBridgeConfig(conf *localtypes.PluginConf) error {
+	if conf.Bridge == "" {
+		conf.Bridge = DefaultBridge
+	}
+	allowedBridgeNames := strings.Split(conf.Bridge, ",")
+	for i := range allowedBridgeNames {
+		allowedBridgeNames[i] = strings.TrimSpace(allowedBridgeNames[i])
+	}
+
+	pfLink, err := c.netlink.LinkByName(conf.PFName)
```

Comment:
I assume the PfName was already exist in our conf so it will hard to change it to uplink right?

Reply:
is that to support SF in the future ?

Reply:
Yes, PFName already exist as a part of internal state. This field is not user-visible, but we can't change its name because it will break backward compatibility and we will not be able to shutdown PODs which were created with old accbridge CNI version. let's keep it as is